Formula/ungit.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class Ungit < Formula
+  desc "The easiest way to use git. On any platform. Anywhere."
+  homepage "https://github.com/FredrikNoren/ungit"
+  url "https://registry.npmjs.org/ungit/-/ungit-1.4.36.tgz"
+  sha256 "3bb07b99e2616afd3059809768036eaefd92d1a62cfab9dc087b9405933ec7f4"
+  head "https://github.com/FredrikNoren/ungit.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"ungit", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
As I newly commented, this app is an embedded web server against a git repo. So, I think the tests are non-trivial to execute. Plus, that's beyond my rudimentary ruby skills. :(

Reply:
You could have a look at the `httpd` formula which spawns a server that is then tested - it might give you enough clues to pull it off.

Reply:
I implemented a live httpd test. Please take a look. Thanks!

Reply:
Looks like audit.rb failed w/
`Error: undefined local variable or method 'curl_openssl_or_deps'`